cmd/satellite/open_stream.go

```diff
@@ -15,17 +15,14 @@
 package satellite
 
 import (
-	"errors"
```

Comment:
Pattern in this repo is to keep first party imports grouped at the top, then third party imports below.